data/pages.yml

```diff
@@ -200,6 +200,8 @@
       url: "configuring-your-app"
     - title: "Configuring Ember CLI"
       url: "configuring-ember-cli"
+    - title: "Preparing for Ember Upgrades"
+      url: "preparing-for-upgrade"
```

Comment:
Is it intended that more material will be added to this section later? The PR and content call it a "deprecation guide", but the title of the section indicates a much broader topic. I've written a few posts that go beyond just deprecations...maybe we can work some more material in to round out the section (in a later PR, perhaps)

[http://jhawk.co/upgrading-ember-i]
[http://jhawk.co/upgrading-ember-ii]
[http://jhawk.co/upgrading-ember-iii]